src/main/gameFiles.ts

```diff
@@ -40,3 +40,18 @@ export async function deleteGameFiles(
     await fs.promises.unlink(`${skyrimDirectory}/${file}`);
   }
 }
+
+export async function checkGameFiles(
+  modDirectory: string,
+  skyrimDirectory: string
+) {
+  logger.info("Checking game files");
+  const gameFolderFilesDirectory = `${modDirectory}/Game Folder Files/`;
+  const gameFilesWithoutEnb = await getFilesWithoutEnb(
+    gameFolderFilesDirectory
+  );
+  const existingGameFilesWithoutEnb = gameFilesWithoutEnb.filter((file) =>
+    fs.existsSync(`${skyrimDirectory}/${file}`)
+  );
+  return existingGameFilesWithoutEnb === gameFilesWithoutEnb;
+}
```

Comment:
Minor tweak but might be worth clarifying what is being checked.
```suggestion
export async function checkGameFilesExist(
  modDirectory: string,
  skyrimDirectory: string
) {
  logger.info(`Checking which game files exist in ${skyrimDirectory}`);
  const gameFolderFilesDirectory = `${modDirectory}/Game Folder Files/`;
  const gameFilesWithoutEnb = await getFilesWithoutEnb(
    gameFolderFilesDirectory
  );
  const existingGameFilesWithoutEnb = gameFilesWithoutEnb.filter((file) =>
    fs.existsSync(`${skyrimDirectory}/${file}`)
  );
  return existingGameFilesWithoutEnb === gameFilesWithoutEnb;
}
```